README.md

```diff
@@ -58,56 +58,54 @@ import KlaviyoSwift
 2. To add Klaviyo's tracking functionality, include the following line in AppDelegate.swift, within `application:didFinishLaunchingWithOptions`:
```

Comment:
I can't comment on the actual line in the README, since it wasn't changed, but should we take out the part that says this is in a closed beta? Or edit it to "open" beta